gothemis/session/session.go

```diff
@@ -237,28 +250,28 @@ func (ss *SecureSession) Unwrap(data []byte) ([]byte, bool, error) {
 	case (C.GOTHEMIS_SUCCESS == res) && (0 < outLen):
 		return out, false, nil
 	case (C.GOTHEMIS_SSESSION_GET_PUB_FOR_ID_ERROR == res):
-		return nil, false, errors.NewCallbackError("Failed to unwrap data (get_public_key_by_id callback error)")
+		return nil, false, ErrCallbackUnwrapData
 	}
 
-	return nil, false, errors.New("Failed to unwrap data")
+	return nil, false, ErrUnwrapData
 }
 
 // GetRemoteID returns ID of the remote peer.
 func (ss *SecureSession) GetRemoteID() ([]byte, error) {
 	// secure_session_get_remote_id
 	var outLength C.size_t
 	if C.secure_session_get_remote_id(ss.ctx.session, nil, &outLength) != C.THEMIS_BUFFER_TOO_SMALL {
-		return nil, errors.NewCallbackError("Failed to get session remote id length")
+		return nil, ErrCallbackGetRemoteIDLen
 	}
```

Comment:
No idea why this is made a "callback error", but GetRemoteID is not a callback. So these errors can be `ErrRemoteIDLength` and `ErrRemoteID` or something.